xcs-ui/xcs-ui-common/src/main/resources/FlamingoThemes/XCS-Blue.xml

```diff
@@ -1010,10 +1008,7 @@ table.xwiki-livetable th.xwiki-livetable-display-header-text * {
 font-weight: bold;
 }
 table.xwiki-livetable td.xwiki-livetable-display-header-filter {
-padding-bottom: @td-livetable-padd-btm !important;
-}
-body.skin-flamingo .xwiki-livetable-display-header tr th.xwiki-livetable-display-header-text{
-background-color: @th-livetable-bg;
+padding: @td-livetable-padd !important;
```

Comment:
We don't encourage the use of '!important' values since they will be hard to over specify.